java/arcs/core/entity/VariableEntityBase.kt

```diff
@@ -0,0 +1,41 @@
+package arcs.core.entity
+
+import arcs.core.common.Referencable
+import arcs.core.data.FieldName
+import arcs.core.data.RawEntity
+import arcs.core.data.Schema
+
+open class VariableEntityBase(
```

Comment:
Comment would be very useful here. Please say both what it does and why, i.e. when it is used.